--- /usr/share/X11/xorg.conf.d/nvidia.conf	2020-03-11 09:56:51.000000000 +0000
+++ /etc/X11/xorg.conf.d/nvidia.conf	2020-04-13 20:17:29.089525340 +0000
@@ -1,3 +1,5 @@
+#fix for connecting external monitor to Optimus NVIDIA GPU
+#Option "PrimaryGPU" "yes": forces use of NVIDIA GPU always
 #This file is provided by xorg-x11-drv-nvidia
 #Do not edit
 
@@ -8,6 +10,7 @@
 	Option "AllowEmptyInitialConfiguration"
 	Option "SLI" "Auto"
 	Option "BaseMosaic" "on"
+	Option "PrimaryGPU" "yes"
 EndSection
 
 Section "ServerLayout"
